lib/content_block_tools/presenters/block_presenters/contact/telephone_presenter.rb

```diff
@@ -71,12 +71,10 @@ def bsl_details
             render_govspeak(bsl_guidance[:value], root_class: "govuk-!-margin-bottom-0") if bsl_guidance[:show]
           end
 
-          def opening_hours_list
-            content_tag(:ul) do
-              item[:opening_hours].each do |item|
-                concat opening_hours_list_item(item)
-              end
-            end
+          def opening_hours
+            opening_hours = item[:opening_hours] || {}
+
+            render_govspeak(opening_hours[:opening_hours], root_class: "govuk-!-margin-bottom-0") if opening_hours[:show_opening_hours]
```

Comment:
are we expecting that the opening hours will be described with govspeak markup? If so, should we update the test in the Content Block Manager and the example in Publishing API?

Or are we using `render_govspeak` to apply a zero margin-bottom wrapper?